src/core/mac/mac_frame.hpp

```diff
@@ -1206,6 +1206,8 @@ class RxFrame : public Frame
 
     /**
      * This method returns the timestamp when the frame was received.
+     * If CSL or TIME_SYNC is enabled, the timestamp marks the frame detection time: the end of the last symbol of SFD.
+     * If neither CSL nor TIME_SYNC are enabled, it may also mark the end of the last symbol of the frame.
```

Comment:
Do we actually use a timestamp that marks the end of the frame? Why not always mark end of SFD?

Reply:
Given the timestamp of the end of frame, it should be possible to compute the timestamp of the SFD as well, right?

Reply:
Yes, certainly. Though for a simple platform not supporting CSL this would not be needed; it then adds extra code which would have no perceivable effect . But to avoid confusion it's also fine to be clear that timestamp is SFD time. E.g. like 

```
* The timestamp marks the frame detection time: the end of the last symbol of SFD.
* The accuracy of the timestamp is platform-dependent.
```

Maybe go for that?

Reply:
Yes, I think it is simpler if this timestamp always has the same meaning, regardless of platform capabiltiies.

I think we can simplify to just the first line:
```
* The timestamp marks the frame detection time: the end of the last symbol of SFD.
```

Reply:
Ok, updated now!